stories/atoms/components/button/Button.stories.mdx

```diff
@@ -29,27 +27,26 @@ import { Icon } from 'ui/atoms/icon/Icon'
 
 # Button
 
-> Communicate _actions_ that users can take.
+> Communicate actions that users can take.
 
 [![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
 
-## Overview
+## Description
 
-### Description
+Buttons are mainly used to give the user options to act. They help users to find
+the most important actions throughout UI and enables them to perform these actions.
 
-Buttons are mainly used to show the user’s choice of options for actions. A button helps the users to find
-the most important actions throughout UI and enables them to perform these actions.  
-By using one or more of the **content** properties it's possible to create appealing and intuitive buttons
+By using one or more of the button properties it's possible to create appealing and intuitive buttons
 that will help users to find the action they're looking for.
 
----
+## Overview
 
 <Story
   name="Overview"
   argTypes={{
-    icon: { control: false, table: { type: { summary: 'JSX.Element' } } },
-    leftIcon: { control: false, table: { type: { summary: 'JSX.Element' } } },
-    rightIcon: { control: false, table: { type: { summary: 'JSX.Element' } } }
+    icon: { control: false, table: { type: { summary: 'Node' } } },
```

Comment:
Regarding the `icon` property, I can't see a dedicated story to explain how it works..